### DISCUSSION TITLE
Where is the documentation?

### DISCUSSION THREAD

**Discussion by galache** [PrismLibrary/Prism#3093] [WPF]
Since 02/27/204 some documentation links have stopped working, such as https://prismlibrary.com/docs/wpf/dialog-service.html.
Where could I find the documentation that was available the day before?
Thank you
Jose Miguel

**Comment by brianlagunas** [comment]
The link is on the website: https://docs.prismlibrary.com/

**Reply by dansiegel** [reply]
Also keep in mind that the docs have been updated for Prism 9.0. As part of the Prism 9.0 milestone the interfaces for Dialogs and Regions were migrated to the Prism.Core. This means that the docs site has been simplified to discuss topics such as Dialogs and Regions have been consolidated since the API is the same regardless of which platform you are developing for. If you have external links to the docs site those docs are likely to have been moved.

**Reply by galache** [reply]
Thanks for the reply. 
Does that mean that the documentation for version 8.0 disappears?
